### DISCUSSION TITLE
Sending email via third-party smtp ( sparkpost )

### DISCUSSION THREAD

**Discussion by tiptap-zz** [directus/directus#5824] [Archive]
I'm trying to configure directus to send via Sparkpost's SMTP services, my .env is as follows;

EMAIL_FROM="no-reply@domain.com"
EMAIL_TRANSPORT="smtp"
EMAIL_SMTP_HOST="smtp.sparkpostmail.com"
EMAIL_SMTP_PORT="587"
EMAIL_SMTP_POOL:"false"
EMAIL_SMTP_USER="SMTP_Injection"
EMAIL_SMTP_PASSWORD="password"
EMAIL_SMTP_SECURE="true"

The above works when sending through curl on my terminal but in directus I get the following error;

[Email] Unexpected error while sending an email:
140434969397120:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:

I have a lets encrypt ssl on the server, but I don't think the ssl on my end would be a problem?

Any ideas how to get this working?

Thanks

**Comment by rijkvanzanten** [comment]
Seems like there's a couple known things over on the nodemailer project: https://github.com/nodemailer/nodemailer/issues/165

I'll have to double check if those options can all be configured through env vars yet in directus. Get back to you on that later (on mobile rn)

**Reply by rijkvanzanten** [reply]
@tiptap-zz Have you tried using port 465 instead of 587? That seems to be a recurring theme under `nodemailer`. If that doesn't help, we can look into what setting needs to be tweaked in nodemailer and expose that through an env var in Directus 👍🏻

**Reply by tiptap-zz** [reply]
sparkpost isnt available on that port. its either 587 or 2525.....

I just tried port 2525 - same error

**Reply by rijkvanzanten** [reply]
Would you mind running some tests using `nodemailer` directly to see what configuration is required by sparkpost? I have no experience with the service

**Reply by tiptap-zz** [reply]
I'll have test with nodemailer, but for reference this is their required settngs for smtp;

Host smtp.sparkpostmail.com
Port 587
Alternative Port 2525
Authentication AUTH LOGIN
Encryption STARTTLS
Username SMTP_Injection
Password The api id

**Reply by tiptap-zz** [reply]
sorted!

needed to set EMAIL_SMTP_SECURE to false as that uses STARTTLS
